packages/content/templates/nuxt-content.js

```diff
@@ -37,6 +38,8 @@ function propsToData (node, doc) {
       obj[field] = evalInContext(value, doc)
       data.on = data.on || {}
       data.on[event] = e => doc[value] = processor(e)
+    } else if (rxClass.test(key)) {
+      data.class = value
```

Comment:
CSS classes are supposed to be set at root level, according to [Vue documentation](https://vuejs.org/v2/guide/render-function.html#The-Data-Object-In-Depth), not as `data.attrs.class`.